src/librustc/session/config.rs

```diff
@@ -284,6 +285,21 @@ impl Externs {
     }
 }
 
+// Similar to 'Externs', but used for the '--extern-private' option
```

Comment:
It would be better to merge this into a single table with `Externs` with the privacy flag being a part of the value.